.github/workflows/wasmcloud.yml

```diff
@@ -216,9 +283,149 @@ jobs:
+          - name: keyvalue-vault
+            target: x86_64-pc-windows-gnu
+          - name: keyvalue-vault
+            target: x86_64-unknown-linux-musl
+
+          - name: http-client
+            target: aarch64-apple-darwin
+          - name: http-client
+            target: aarch64-unknown-linux-musl
+          - name: http-client
+            target: x86_64-apple-darwin
+          - name: http-client
+            target: x86_64-pc-windows-gnu
+          - name: http-client
+            target: x86_64-unknown-linux-musl
+
+          - name: http-server
+            target: aarch64-apple-darwin
+          - name: http-server
+            target: aarch64-unknown-linux-musl
+          - name: http-server
+            target: x86_64-apple-darwin
+          - name: http-server
+            target: x86_64-pc-windows-gnu
+          - name: http-server
+            target: x86_64-unknown-linux-musl
+
+          - name: messaging-kafka
+            target: aarch64-apple-darwin
+          - name: messaging-kafka
+            target: aarch64-unknown-linux-musl
+          - name: messaging-kafka
+            target: x86_64-apple-darwin
+          - name: messaging-kafka
+            target: x86_64-pc-windows-gnu
+          - name: messaging-kafka
+            target: x86_64-unknown-linux-musl
+
+          - name: messaging-nats
+            target: aarch64-apple-darwin
+          - name: messaging-nats
+            target: aarch64-unknown-linux-musl
+          - name: messaging-nats
+            target: x86_64-apple-darwin
+          - name: messaging-nats
+            target: x86_64-pc-windows-gnu
+          - name: messaging-nats
+            target: x86_64-unknown-linux-musl
+
+          - name: sqldb-postgres
+            target: aarch64-apple-darwin
+          - name: sqldb-postgres
+            target: aarch64-unknown-linux-musl
+          - name: sqldb-postgres
+            target: x86_64-apple-darwin
+          - name: sqldb-postgres
+            target: x86_64-pc-windows-gnu
+          - name: sqldb-postgres
+            target: x86_64-unknown-linux-musl
+
+    name: ${{ matrix.config.name }}-provider-${{ matrix.config.target }}
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+      - uses: ./.github/actions/install-nix
+        # need to run condition inside job steps so that job will pass if all steps are skipped
+        # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpull_requestpull_request_targetbranchesbranches-ignore
+        with:
+          cachixAuthToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+      - uses: ./.github/actions/build-nix
+        with:
+          package: ${{ matrix.config.name }}-provider-${{ matrix.config.target }}
+
+  build-wash-windows:
     if: ${{ startswith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/main'}}
-    name: wasmcloud-x86_64-pc-windows-msvc
+    name: wash-x86_64-pc-windows-msvc
```

Comment:
Should we have a followup to build wash for `gnu` instead of `msvc` like we're doing for wasmCloud?

Reply:
As mentioned in the comment we cannot do that due to https://github.com/rust-lang/rust/issues/92212. This has been a problem since 2023 (https://github.com/wasmCloud/wasmCloud/pull/807)

The only way this would work today is if `wash` binary would be smaller and had fewer dependencies, which is unlikely to happen soon. The other option is to either wait for either https://github.com/rust-lang/rust/issues/92212 to be fixed or use some other target (like `pc-windows-gnullvm` https://doc.rust-lang.org/rustc/platform-support/pc-windows-gnullvm.html)